src/App.jsx

```diff
@@ -1,44 +1,12 @@
 import React from 'react';
+import { Article, firstArticle } from './components/Article';
```

Comment:
I think you shouldn't import the firstArticle object because this object should be in this component

Reply:
I don't like both options because components must be separated if I understood correctly 
I made a personal file for firstArticle, but I would like to hear the mentor's opinion in this case

Reply:
I agree with you the components must be separated but firstArticle is not a component is an object with data